File: app/src/main/java/org/nervos/neuron/util/sensor/SensorIDRandomUtils.java
package org.nervos.neuron.util.sensor;

/**
 * Created by BaojunCZ on 2018/10/12.
 */
public class SensorIDRandomUtils {

    public static String getID() {
        String x = "";
        for (int n = 0; n < 51; n++) {
            x += (int) (10 * (Math.random()));
        }
        x += System.currentTimeMillis();
        return x;
    }

}

File: app/src/main/java/org/nervos/neuron/util/sensor/SensorIDRandomUtils.kt
package org.nervos.neuron.util.sensor

/**
 * Created by BaojunCZ on 2018/10/12.
 */
object SensorIDRandomUtils {

    val id: String
        get() {
            var x = ""
            for (n in 0..50) {
                x += (10 * Math.random()).toInt()
            }
            x += System.currentTimeMillis()
            return x
        }

    val ipId:String
        get() {
            var x = ""
            for (n in 0..63) {
                x += (10 * Math.random()).toInt()
            }
            return x
        }

}
